prover/src/utils.rs

```diff
@@ -0,0 +1,90 @@
+use lambdaworks_math::field::{
+    element::FieldElement, fields::fft_friendly::babybear_u32::Babybear31PrimeField,
+};
+
+type FE = FieldElement<Babybear31PrimeField>;
+
+pub fn u32_to_2_limbs(x: u32) -> [FE; 2] {
+    [x & 0xFFFF, (x >> 16) & 0xFFFF].map(FE::new)
+}
+
+pub fn u32_to_4_limbs(x: u32) -> [FE; 4] {
+    [
+        x & 0xFF,
+        (x >> 8) & 0xFF,
+        (x >> 16) & 0xFF,
+        (x >> 24) & 0xFF,
+    ]
+    .map(FE::new)
+}
+
+// TODO: Revisar la logica de como guardar integers in la tabla.
```

Comment:
```suggestion
// TODO: Review logic that stores integers in the table.
```

Also, does that logic refer to an already implemented function in this repo?

Reply:
Lambdaworks does not support creating a field element directly from an integer. I checked the logic and updated this function accordingly.